diff --git a/mlir/python/CMakeLists.txt b/mlir/python/CMakeLists.txt
--- a/mlir/python/CMakeLists.txt
+++ b/mlir/python/CMakeLists.txt
@@ -876,6 +876,8 @@ add_mlir_python_common_capi_library(MLIR
     ${_ADDL_TEST_SOURCES}
 )
 
+file(MAKE_DIRECTORY "${MLIR_BINARY_DIR}/python_packages/mlir_core/mlir/_mlir_libs")
+
 ################################################################################
 # Custom targets.
 ################################################################################
